crates/proto/src/rr/domain/label.rs

```diff
@@ -37,6 +37,9 @@ impl Label {
     /// This uses the bytes as raw ascii values, with nothing escaped on the wire.
     /// Generally users should use `from_str` or `from_ascii`
     pub fn from_raw_bytes(bytes: &[u8]) -> ProtoResult<Self> {
+        if bytes.is_empty() {
+            return Err("Label requires a minimum length of 1".into());
```

Comment:
Would be nice to add a comment here referring to the RFC section.

Reply:
I added the RFC reference and cited the relevant sentence.